Js

VeronikaSoshenko opened the pull request:
Заменили текст меката в хэдере на текст в js

VeronikaSoshenko committed:
verstka

VeronikaSoshenko committed:
Update app.component.html

Удалили поля

VeronikaSoshenko committed:
js

Поправила макет: хэдер корзина, все карточки съехавшие выстроила по размерам, стрелки поставила на место.

pocahontassss commented on src/app/app.component.scss:
```diff
@@ -0,0 +1,764 @@
+  background-color:#4B565E;
+  color: white;
+  padding: 0 240px;
+  font-family: Inter;
```
Вместо того, что бы в каждом классе прописывать font-family: Inter, мы задаем элементу body это свойство, благодаря чему каждый текстовый элемент уже будет иметь свойство font-family: Inter

pocahontassss commented on src/app/app.component.scss:
```diff
@@ -0,0 +1,764 @@
+  text-decoration: none;
+  margin: auto;
+
+}
```
Лишняя пустая строка на 42

pocahontassss commented on src/app/app.component.scss:
```diff
@@ -0,0 +1,764 @@
+color: white;
+border-color: #F0BA4E;
+display: flex;
+}
```
Поплыла табуляция

pocahontassss commented on src/app/app.component.scss:
```diff
@@ -0,0 +1,764 @@
+margin-left: 10px;
+margin-top: 5px;
+margin-right: 10px;
+}
```
Здесь тоже поплыла

pocahontassss commented on src/app/app.component.scss:
```diff
@@ -0,0 +1,764 @@
+    box-shadow: 0px 20px 50px 0px #3C3C3B12;
+    max-height: 428px;
+    max-width: 421px;
+    background-image: url(/assets/image19.png);
```
Ссылки в scss указываем так:
`url('/assets/image19.png');`
То-есть нужно поставить кавычки, когда указываешь путь к файлу

pocahontassss commented on src/app/app.component.scss:
```diff
@@ -0,0 +1,764 @@
+    background: none;
+    color: #A1A0A0 ;
+      
+    } 
```
Нет табуляции + лишняя строка на 141

pocahontassss commented on src/app/app.component.scss:
```diff
@@ -0,0 +1,764 @@
+  .item-cards_grid-left-action-link {
+width: 40px;
+height: 40px;
+  }
```
И здесь тоже поплыла

pocahontassss commented on src/app/app.component.scss:
```diff
@@ -0,0 +1,764 @@
+  top: 318px;
+  left: 60px;
+  padding: 10px 10px 10px 10px;
+  border-radius: 36px 36px 36px 36px;
```
Можно указать просто border-radius: 36px и оно применится для всех сторон

pocahontassss commented on src/app/app.component.scss:
```diff
@@ -0,0 +1,764 @@
+}
+
+.infocard_tocard-action {
+  padding: 14px 28px 14px 28px;
```
Можно указать padding: 14px 28px (первое свойство для верха и низа, второе для лево и право)

pocahontassss commented on src/app/app.component.scss:
```diff
@@ -0,0 +1,764 @@
+  margin: 0;
+  appearance: none;
+  outline: none;
+}
```
Для миксинов лучше создать файл normalize.scss и импортировать его сюда через @import

pocahontassss commented on src/assets/Facebook.svg:
Все названия картинок пишем с маленькой буквы и нужно поработать над названием. Так же картинки желательно именовать всегда на английском языке, русский не используем.
Подробнее здесь:
https://docs.google.com/document/u/0/d/1ipRxAtW0KEC5bj-vWLNk-p8AwjsN0AWmN8YApyP9VJw/mobilebasic
(Внизу информация про наименование, а так желательно почитать всю статью)

pocahontassss commented on src/app/app.component.html:
```diff
@@ -1,336 +1,395 @@
+    <div class="bar-logo">
+      <img src="/assets/menu.svg" alt="catalogue icon"> 
+    </div>
+    <nav class="bar-nav">
+      <div class="bar-nav-item">{{BarItem1}}</div>
+      <div class="bar-nav-item">{{BarItem2}}</div>
+      <div class="bar-nav-item">{{BarItem3}}</div>
+      <div class="bar-nav-item">{{BarItem4}}</div>
+      <div class="bar-nav-item">{{BarItem5}}</div>
+    </nav>
```
Это нужно сделать кнопками, так как эти кнопки открывают доп. меню

pocahontassss commented on src/app/app.component.html:
```diff
@@ -1,336 +1,395 @@
+          </div>
+          <div class="item-cards_grid-right-block2">
+            <div class="item-cards_grid-right-block-text2">
+              Лакокрасочные <br> материалы
```
Тег br мы используем очень редко, здесь лучше ограничить ширину контейнера и текст сам перейдет на новую строку

pocahontassss commented on src/app/app.component.html:
```diff
@@ -1,336 +1,395 @@
+                <div class="infocard3">
+                  <h5 style="font-size: 16px;font-weight: 500;text-align: left; margin-top: 0;margin-bottom: 5px;">Количество</h5>
+                  <button type="button" class="infocard3-addtocart">-</button>
+                  <span style="font-family: Inter;font-size: 20px;font-weight: 500;margin-right: 8px;margin-left: 8px;">1</span>
+                  <button type="button" class="infocard3-addtocart">+</button>
+                  <button type="button" class="infocard_tocard-action">В корзину</button>
+                </div>
+          </div>
+      </li>
+      <li class="popular_product-card">
+        <div class="infocard-text">
+          <div class="infocard1"> 
+            <h4 style="font-size: 20px;font-weight: 600;text-align: left; margin-bottom: 10px;"> Круг зачистной 125х6х22 </h4>
+            <h5 style="font-size: 16px;font-weight: 500; text-align: left;margin: 0">Артикул 00004 </h5>
+          </div>
+        <div class="infocard2">
+            <h5 style="font-size: 16px;font-weight: 500;text-align: left;color: #A1A0A0;margin-top: 20px; margin-bottom: 5px;">Цена за штуку</h5>
+            <h6 style="font-size: 16px;font-weight: 600;text-align: left; margin-top: 0;"> <s>52 руб.</s> <b style="font-size: 20px;font-weight: 600;text-align: left;margin-top: 0; margin-bottom: 0;"> 48 руб.</b></h6>
+        </div>
+        <div class="infocard3">
+  <h5 style="font-size: 16px;font-weight: 500;text-align: left; margin-top: 0;margin-bottom: 5px;">Количество</h5>
+  <button type="button" class="infocard3-addtocart">-</button>
+  <span style="font-family: Inter;font-size: 20px;font-weight: 500;margin-right: 8px;margin-left: 8px;">1</span>
+  <button type="button" class="infocard3-addtocart">+</button>
+  <button type="button" class="infocard_tocard-action">В корзину</button>
+        </div>
+        </div>
+      </li>
+      <li class="popular_product-card">
+        <div class="infocard-text">
+          <div class="infocard1"> 
+            <h4 style="font-size: 20px;font-weight: 600;text-align: left; margin-bottom: 10px;"> Круг зачистной 125х6х22 </h4>
+            <h5 style="font-size: 16px;font-weight: 500; text-align: left;margin: 0">Артикул 00004 </h5>
+          </div>
+        <div class="infocard2">
+            <h5 style="font-size: 16px;font-weight: 500;text-align: left;color: #A1A0A0;margin-top: 20px; margin-bottom: 5px;">Цена за штуку</h5>
+            <h6 style="font-size: 16px;font-weight: 600;text-align: left; margin-top: 0;"> <s>52 руб.</s> <b style="font-size: 20px;font-weight: 600;text-align: left;margin-top: 0; margin-bottom: 0;"> 48 руб.</b></h6>
+        </div>
+        <div class="infocard3">
+  <h5 style="font-size: 16px;font-weight: 500;text-align: left; margin-top: 0;margin-bottom: 5px;">Количество</h5>
```
Слишком много стилей, лучше выносить в классы) Код становится нечитабельным

pocahontassss commented on src/app/app.component.html:
```diff
@@ -1,336 +1,395 @@
+<section>
+  <div class="catalogue-action">
+<img style="float: right;" src="/assets/pngwing1.svg" alt="man">
+<div class="catalogue-text">
+<h3 style="font-family: Inter;font-size: 28px;font-weight: 500;text-align: left;color: #3C3C3B; padding-top: 28px; padding-left: 28px;">Более 10 000 наименований </h3>
+<h4 style="font-family: Inter; font-size: 20px; font-weight: 400; text-align: left; color: #A1A0A0; padding-top: 28px; padding-left: 28px;">Инструмент для вашего бизнеса, разнообразных марок</h4>
+<a style="font-family: Inter;font-size: 18px; font-weight: 600; padding-top: 28px; padding-left: 28px; color: #F0BA4E; text-align: left; margin-top: 20px;">Перейти в каталог</a>
+</div>
```
Аналогично комментарию выше, код плохо читается, выносим в классы

pocahontassss commented on src/app/app.component.html:
```diff
@@ -1,336 +1,395 @@
+               <h5 style="font-size: 16px;font-weight: 500;text-align: left;color: #A1A0A0;margin-top: 20px; margin-bottom: 5px;">Цена за штуку</h5>
+               <b style="font-size: 20px;font-weight: 600;text-align: left;margin-top: 0; margin-bottom: 0;"> 52 руб.</b>
+              </div>
+              <div class="infocard3">
+        <h5 style="font-size: 16px;font-weight: 500;text-align: left; margin-top: 37px;margin-bottom: 5px;">Количество</h5>
+                  <button type="button" class="infocard3-addtocart">-</button>
+                  <span style="font-family: Inter;font-size: 20px;font-weight: 500;margin-right: 8px;margin-left: 8px;">1</span>
+                  <button type="button" class="infocard3-addtocart">+</button>
+                  <button type="button" class="infocard_tocard-action">В корзину</button>
+                </div>
```
Поплыла табуляция на 268

pocahontassss commented on src/app/app.component.html:
```diff
@@ -1,336 +1,395 @@
+            <h5 style="font-size: 16px;font-weight: 500;text-align: left;color: #A1A0A0;margin-top: 20px; margin-bottom: 5px;">Цена за штуку</h5>
+            <b style="font-size: 20px;font-weight: 600;text-align: left;margin-top: 0; margin-bottom: 0;"> 52 руб.</b>
+        </div>
+        <div class="infocard3">
+  <h5 style="font-size: 16px;font-weight: 500;text-align: left; margin-top: 37px;margin-bottom: 5px;">Количество</h5>
+  <button type="button" class="infocard3-addtocart">-</button>
+  <span style="font-family: Inter;font-size: 20px;font-weight: 500;margin-right: 8px;margin-left: 8px;">1</span>
+  <button type="button" class="infocard3-addtocart">+</button>
+  <button type="button" class="infocard_tocard-action">В корзину</button>
+        </div>
```
Поплыла табуляция

pocahontassss commented on src/app/app.component.html:
```diff
@@ -1,336 +1,395 @@
+            <h5 style="font-size: 16px;font-weight: 500;text-align: left;color: #A1A0A0;margin-top: 20px; margin-bottom: 5px;">Цена за штуку</h5>
+             <b style="font-size: 20px;font-weight: 600;text-align: left;margin-top: 0; margin-bottom: 0;"> 52 руб.</b>
+        </div>
+        <div class="infocard3">
+  <h5 style="font-size: 16px;font-weight: 500;text-align: left; margin-top: 37px;margin-bottom: 5px;">Количество</h5>
+  <button type="button" class="infocard3-addtocart">-</button>
+  <span style="font-family: Inter;font-size: 20px;font-weight: 500;margin-right: 8px;margin-left: 8px;">1</span>
+  <button type="button" class="infocard3-addtocart">+</button>
+  <button type="button" class="infocard_tocard-action">В корзину</button>
+        </div>
```
Снова поплыла

pocahontassss commented on src/app/app.component.html:
```diff
@@ -1,336 +1,395 @@
+            <img src="/assets/call_Icon.svg" class="footer _contacts-image">
+            </div>
+              <div class="footer_contacts-content">
+                <span class="footer_contacts-title">Контактный телефон</span>
+                <span class="footer_contacts-phone">+7(812)309-09-344 </span>
+                <span class="footer_contacts-phone">+7(960)84-29-09</span>
```
Верхний номер правильно сделала, ссылкой с href= tel+, здесь так-же нужно.

pocahontassss commented on src/app/app.component.html:
```diff
@@ -1,336 +1,395 @@
+              <img src="/assets/mail_icon.svg" class="footer_contacts-image">
+            </div>
+          <div class="footer_contacts-content">
+            <span class="footer_contacts-title"> Электронная почта</span>
+            <span class="footer_contacts-mail">tustin78.mail.ru</span>
+            <span class="footer_contacts-mail">info.tustin.ru </span>
```
Почту нужно сделать тоже ссылками с использованием href=mailto

pocahontassss commented on src/app/app.component.html:
```diff
@@ -1,336 +1,395 @@
-          </a>
-        }
+  <div class="footer_email">
+    <div class="footer_title-wrapper">
+      <h3 class="footer_title"> Остались: вопросы? </h3>
+      <p class="footer_label"> Узнавайте первым о скидках, предложениях и событиях еженедельно в своем почтовом ящике. Отпишитесь в любое время одним щелчком мыши.</p>
+        <form class="footer_form-wrapper">
+          <img src="/assets/mail_icon.svg" class="footer_input-icon">
+          <input type="email" placeholder="Введите свою почту" class="footer_input">
+          <button type="submit" class="footer_email-submit"> Отправить </button>
+        </form>
+        </div>
```
Поплыла табуляция

VeronikaSoshenko committed:
Дз Js

Исправила в коде мелкие ошибки, выявленные в прошлом код ревью. Перенесла стили прописанные в html в css. Выполнила домашние задания 1-3, 4 тоже, но не понимаю почему оно не работает
